pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -83,8 +86,8 @@ func writeVifFile(buf []byte, pid, name string) error {
 	return nil
 }
 
-func setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, uid string) error {
-	cache := PodCacheInterface{Iface: iface}
+func setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, vmi *v1.VirtualMachineInstance, cacheFactory cache.InterfaceCacheFactory) error {
+	cache := &cache.PodCacheInterface{Iface: iface}
```

Comment:
You're shadowing an imported pkg name here.

Reply:
Fixed.